init.sh

```diff
@@ -13,6 +13,8 @@ git clone --depth 1 https://github.com/canonical/sphinx-docs-starter-pack "$temp
 rm -rf "$temp_directory/.git"
 
 # Update file contents for the install directory
+echo "Removing internal workflow files..."
```

Comment:
Since we don't require using the init script, can you also add this to the readme? It needs to go both in the section for a [stand-alone doc repo](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/readme.rst#standalone-documentation-repository) and the one for [docs in the code repo](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/readme.rst#documentation-in-a-code-repository).